src/routes/PlayQuiz/QuestionsList.tsx

```diff
@@ -41,6 +50,20 @@ export default function QuestionsList({
     }
   }
 
+  function getBadgeColor(points) {
+    if (parseInt(points) > minQuestionPoint + questionPointsAverage * 0.8) {
```

Comment:
It should be range instead of questionPointsAverage

Reply:
We should be calculating the color in the range of the values. It has nothing to do with the average.

Reply:
Example: points are 1, 2, 3, 4, 5
Now only 5 should be red. 4 should be orange. But if you calculate average = 3. (min + avg * 0.8) = 3.4 and as per this logic both 4 and 5 is red.

Reply:
Instead range = 4 and (min + range * 0.8) = 4.2; So, only 5 will be red which is the right value.